xml/FrameworksIndex/net-6.0.xml

```diff
@@ -85,7 +86,6 @@
     <Assembly Name="System.Net.NetworkInformation" Version="6.0.0.0" />
     <Assembly Name="System.Net.Ping" Version="6.0.0.0" />
     <Assembly Name="System.Net.Primitives" Version="6.0.0.0" />
-    <Assembly Name="System.Net.Quic" Version="6.0.0.0" />
```

Comment:
@danmoseley Looks like this Preview 7 update did what we wanted.

Reply:
will this magically fix https://github.com/dotnet/dotnet-api-docs/issues/6957 ? @ManickaP

Reply:
I think it will, @danmoseley. If you go to the Files tab and scroll, you'll see the Quic xml files are being deleted.

BTW, these deleted files are causing a conflict with the main branch @gewarren .